webpack.config.node.js

```diff
@@ -81,7 +82,9 @@ const webpackConfig = {
 					plugins: [ [
 						path.join( __dirname, 'server', 'bundler', 'babel', 'babel-plugin-transform-wpcalypso-async' ),
 						{ async: false }
-					] ]
+					] ],
+					cacheDirectory: path.join( __dirname, './build/.babel-cache' ),
```

Comment:
I did that on purpose because I'm not sure of what would happen if they shared a cache...

It looks like the filenames are created by hashing the contents so it _should_ work